code/api/src/modules/shipment/resolvers.js

```diff
@@ -0,0 +1,33 @@
+// App Imports
+import models from '../../setup/models'
+
+// Get all shipments
+export async function getAll() {
+  return await models.Shipment.findAll({ order: [['id', 'DESC']] })
+}
+
+// Get shipment by ID
+export async function getById(parentValue, { id }) {
+  return await models.Shipment.findOne({
+    where: { id },
+    include: [
+      { model: models.Crate, as: 'crate' },
+    ]
+  })
+}
+
+// Get shipments by user
+export async function getByUser(parentValue, { }, { auth }) {
```

Comment:
This looks like it will be perfect for what I need to connect to the shipments component